client/gutenberg/extensions/presets/jetpack/editor.js

```diff
@@ -5,8 +5,19 @@
  */
 import './shared/public-path';
 import './editor-shared/block-category'; // Register the Jetpack category
-import 'gutenberg/extensions/contact-form/editor';
-import 'gutenberg/extensions/markdown/editor';
-import 'gutenberg/extensions/map/editor';
-import 'gutenberg/extensions/publicize/editor';
-import 'gutenberg/extensions/simple-payments/editor';
+
+// TODO: Generate dyanmically from index.json
+// Appending `Block` to the names to keep `Map` from colliding with JS' Map
+import * as ContactFormBlock from 'gutenberg/extensions/contact-form/editor';
```

Comment:
There is more then 1 block found here.

Reply:
@enejb what do you think if we split each sub-block into their own file or even directory?

Much like [`trials`](https://github.com/Automattic/wp-calypso/tree/78367825d7cc6a739b2b9b8644d2cc21ce1306af/client/gutenberg/extensions/trials) & [`trial`](https://github.com/Automattic/wp-calypso/tree/78367825d7cc6a739b2b9b8644d2cc21ce1306af/client/gutenberg/extensions/trial) blocks.

Reply:
(I'm all for that, just didn't have time to do it for this PR, which should work without)

Reply:
@simison Sure we can do that as well.